[KOGITO-9600][KSW-Guides]: Update chrome extension docs (#419)
kiegroup/kogito-docs: serverlessworkflow/modules/ROOT/pages/tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc

. Push the created workflow file to GitHub, or you can go to the GitHub page and click on the workflow file to open in the editor.
+
--
.Serverless Workflow file in GitHub(read-only mode)
image::tooling/serverless-workflow-editor/swf-editor-in-github-readonly.png[]
--

. To change the read-only mode to edit mode, click the pencil icon on the top-right corner of the screen. 
+
--
.Serverless Workflow file in GitHub(edit mode)
image::tooling/serverless-workflow-editor/swf-editor-in-github-edit.png[]
--
+

> . You can switch to the full-screen mode. Also, you can hide the editor and view the source code using *See as source* button.

Reviewer: ```suggestion
. You can switch to the full-screen mode. Also, you can hide the editor and view the source code using the *See as source* button.
```

> . You can switch to the full-screen mode. Also, you can hide the editor and view the source code using the *See as source* button.

Reviewer 2: ```suggestion
. You can switch to the full-screen mode. Also, you can hide the editor and view the source code using the `See as source` button.
```

> . Make changes to the source code, add a commit message. You can either commit the changes to a branch directly or create a new branch for pull request.

Reviewer: ```suggestion
. Make changes to the source code, and add a commit message. You can either commit the changes to a branch directly or create a new branch for pull requests.
```

> . In a pull request you can switch between the original and the changes using the buttons provided.

Reviewer: ```suggestion
. In a pull request, you can switch between the original and the changes using the buttons provided.
```
